src/TraceEvent/Parsers/ClrTraceEventParser.cs

```diff
@@ -844,6 +870,89 @@ internal ClrTraceEventParserState State
                 source.UnregisterEventTemplate(value, 38, ProviderGuid);
             }
         }
+        public event Action<GCDynamicTraceData> GCDynamic
```

Comment:
we should certainly think about how to display these in the Events view since we use it heavily. @brianrob could you please give us an idea what work needs to happen for the Events view to display these dynamic events based on the payload (ie, have them on the left hand side so you can select them just like you do with traditional events)? thanks!

Reply:
Tracking: https://github.com/microsoft/perfview/issues/1930 - we plan to separate the ability to get the dynamic events in the TraceGC and GCStats and in the Events View.